app/api/server/v1/voip/rooms.ts

```diff
@@ -176,7 +232,7 @@ API.v1.addRoute(
 			}
 			const language: string = rcSettings.get('Language') || 'en';
 			const comment = TAPi18n.__('Closed_by_visitor', { lng: language });
-			const closeResult = await LivechatVoip.closeRoom(visitor, room);
+			const closeResult = await LivechatVoip.closeRoom(visitor, room, this.user);
```

Comment:
Where the endpoint get the comments/tags and send to voip service as params?

Reply:
oops